common/server_protocolhandler.cpp

```diff
@@ -762,6 +762,11 @@ Server_ProtocolHandler::cmdCreateGame(const Command_CreateGame &cmd, Server_Room
         if (room->getGamesCreatedByUser(QString::fromStdString(userInfo->name())) >= server->getMaxGamesPerUser())
             return Response::RespContextError;
 
+    if (cmd.join_as_judge() && !server->permitCreateGameAsJudge() &&
```

Comment:
This looks fishy to me.
If `permitCreateGameAsJudge` is true, anyone can create a game as a judge even without the needed "judge" permission level.
If `permitCreateGameAsJudge` is false, only judges can create a game as a judge.

Is this how you meant it to be implemented?
Please add the configuration entry and a short description to servatrice.ini.example

Reply:
That's right.  Some games like Werewolf you need the host to be able to do things, so for servers that want to support this type of thing, the creator of the game can start as a judge for this that game.  It's off by default to prevent abuse on servers that are mostly for MtG.

Reply:
I got your point, and it makes sense. Thank you